page-objects/login-page.ts

```diff
@@ -1,29 +1,73 @@
-// page-objects/login-page.ts
+import { Page, Locator } from 'playwright';
 
 export class LoginPage {
-  page: any;
+  private page: Page;
 
-  constructor(page: any) {
+  // Локаторы для элементов
+  private emailInput = 'input[name="email"]';
+  private passwordInput = 'input[name="password"]';
+  private submitButton = 'button[type="submit"]';
+  private userMenuButton = 'a.user-menu__btn';
+  private welcomeMessage = '#welcome-message';
+
+  constructor(page: Page) {
     this.page = page;
   }
 
-  // Метод для перехода на страницу логина
+  // Переход на страницу логина
   async goToLoginPage() {
-    await this.page.goto('https://example.com/login'); // Замените на ваш URL страницы логина
+    try {
```

Comment:
Why do you add try / catch to each method in page object  class?
Playwright or any other framework will throw an appropriate error with description and action that has failed automatically. You don't need to wrap eery action with try / catch